radicle-cli/examples/workflow/3-issues.md

```diff
@@ -0,0 +1,57 @@
+A project "todo" items is called an *issue*.  They can be inspected and
+modified using the `issue` subcommand.
+
+Let's say the new car you are designing with your peers has a problem with its flux capacitor.
+
+```
+$ rad issue open --title "flux capacitor underpowered" --description "Flux capacitor power requirements exceed current supply" --no-announce
+title: flux capacitor underpowered
+state: open
+tags: []
+assignees: []
+
+Flux capacitor power requirements exceed current supply
+```
+
+The issue is now listed under our project.
+
+```
+$ rad issue list
+b05e945bb63c11bf80320f4e26ad1d1f7c51f755 "flux capacitor underpowered"
+```
+
+Great! Now we've documented the issue for ourselves and others.
+
+Just like with other project management systems, the issue can be assigned to
+others to work on.  This is to ensure work is not duplicated.
+
+Let's assign ourselves to this one.
+
+```
+$ rad assign b05e945bb63c11bf80320f4e26ad1d1f7c51f755 --to did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk
+```
+
+It will now show in the list of issues assigned to us.
+
+```
+$ rad issue list --assigned
+b05e945bb63c11bf80320f4e26ad1d1f7c51f755 "flux capacitor underpowered" did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk
+```
+
+Note: this can always be undone with the `unassign` subcommand.
+
+```
+$ rad unassign b05e945bb63c11bf80320f4e26ad1d1f7c51f755 --from did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk
```

Comment:
I'd remove this block/test because we're just making a note, and don't actually want to unassign ourselves..